definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js

```diff
@@ -1,305 +1,322 @@
 // @flow
 
-import firebase from 'firebase'
-import 'firebase/firestore'
-import app from '@firebase/app'
-import auth from 'firebase/auth'
-import database from 'firebase/database'
+import firebase from "firebase";
```

Comment:
All of these look like changes that I *think* aren't consistent with our current prettier setup.. can you verify?

Reply:
my bad. for some reason my editor is linting/prettifying based on my personal preferences and not using the preferences of this repo. will revert.